### PR TITLE
Pd 1294 scrub all mentions of suggestion tickets from the docs hub

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: static/includes/CreateFeatureRequest.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+To submit a new feature request, click **<i class="fa fa-plus" aria-hidden="true"></i> New Topic** in the top right of any forum screen
+Enter a title that describes the requested improvement or new functionality.
+Change the category tag to **Feature Requests**.
+The body of the post populates a template.
+Describe the **Problem/Justification**, **Impact**, and provide **User Story** for your request.
+Click **<i class="fa fa-plus" aria-hidden="true"></i> Create Topic** to submit the request.
+
+{{< trueimage src="/images/Contribute/FeatureRequest.png" alt="Submit a Feature Request" id="Submit a Feature Request" >}}
+
+To vote for a Feature Request, open that thread use the **Vote** button in the top left corner, next to the title.
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:19]: feature request in this case is not a proper noun, and it is talking about it in a general sense and not the form name, so lower case.
```suggestion
To vote for a feature request, open that thread then click **Vote** at the top left corner, next to the title.
```

File: static/includes/CreateFeatureRequest.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+{{< trueimage src="/images/Contribute/FeatureRequestVote.png" alt="Vote for a Feature Request" id="Vote for a Feature Request" >}}
+
+Each forum user has a limited number of votes that can be cast for items at any given time.
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:23]: Active voice/present tense suggested change
```suggestion
Each forum user can cast a limited number of votes for items at any given time.
```

File: static/includes/CreateFeatureRequest.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+The higher your Trust Level, the more votes you can cast.
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:24]: not a proper noun so lower case
```suggestion
The higher your trust Level, the more votes you can cast.
```

File: static/includes/CreateFeatureRequest.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+The number of votes based on Trust Level is as follows:
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:25]: not a proper noun
```suggestion
The number of votes based on trust level is as follows:
```

File: static/includes/CreateFeatureRequest.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+|-----------|-------------|
+| 0 | 2 |
+| 1 | 4 |
+| 2 | 6 |
+| 3 | 8 |
+| 4 | 10 |
+{{< /truetable >}}
+
+Votes on a topic are kept until either you remove the vote manually or the topic is closed.
+Topics are closed periodically as they are reviewed and either accepted or rejected based on their merit and feasibility.
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:38]:
```suggestion
Topics are closed periodically after they are reviewed and either accepted or rejected based on their merit and feasibility.
```

File: static/includes/CreateFeatureRequest.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+## What is a good feature request?
+
+Before creating a new feature request, it is important to take some time to think through the implementation and user story.
+A good feature request includes details about the functionality being added, if there is a UI component, and a detailed user story about how a TrueNAS user might interact with the said feature.
```
Comment (micjohnson777) [inline static/includes/CreateFeatureRequest.md:43]: not a fan of this phrasing, so what about this?
```suggestion
A good feature request includes details about the functionality requested, if it involves a UI component, and a detailed user story describing how a TrueNAS user might interact with the proposed feature.
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
A few suggested changes.

File: static/includes/FileIssueTicketSCALE.md [hunk 7]
```diff
@@ -6,7 +6,7 @@ You must have [a Jira account](https://id.atlassian.com/signup?continue=https%3A
 If you encounter a bug or other issue while using TrueNAS SCALE, you can report issues in one of two ways:
 
 * Log into Jira and use **Create** to open a new ticket [details]({{< relref "JiraIssueReporting.md" >}}).
-* Use the **How would you rate this page?** ![FeedbackIcon](/images/SCALE/Dashboard/FeedbackIcon.png "Feedback Icon") icon or **File Ticket** option on the **System Settings > General** screen to create a bug report in the [TrueNAS Jira Project](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues).
+* Use the **Send Feedback** ![FeedbackIcon](/images/SCALE/Dashboard/FeedbackIcon.png "Feedback Icon") icon or **File Ticket** option on the **System Settings > General** screen to create a bug report in the [TrueNAS Jira Project](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues).
```
[the remark this answers is not in the document]
Comment (DjP-iX) [reply static/includes/FileIssueTicketSCALE.md:9]: good catch 
```suggestion
* Use the **Send Feedback** ![FeedbackIcon](/images/SCALE/Dashboard/FeedbackIcon.png "Feedback Icon") icon or **File Ticket** option on the **System > General** screen to create a bug report in the [TrueNAS Jira Project](https://ixsystems.atlassian.net/jira/software/c/projects/NAS/issues).
```

**Review by micjohnson777 — APPROVED** [review]
Excellent!

**Pull request merged**